contracts/extensions/test/order_helper.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    addressUtils,
+    chaiSetup,
+    constants,
+    OrderFactory,
+    provider,
+    txDefaults,
+    web3Wrapper,
+} from '@0x/contracts-test-utils';
+import { BlockchainLifecycle } from '@0x/dev-utils';
+import { assetDataUtils, orderHashUtils } from '@0x/order-utils';
+import { SignedOrder } from '@0x/types';
+import * as chai from 'chai';
+
+import { OrderHelperContract } from '../generated-wrappers/order_helper';
+import { artifacts } from '../src/artifacts';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+const blockchainLifecycle = new BlockchainLifecycle(web3Wrapper);
+
+describe('OrderHelper extension', () => {
+    let signedOrder: SignedOrder;
+    let orderFactory: OrderFactory;
+    let contract: OrderHelperContract;
+
+    before(async () => {
+        await blockchainLifecycle.startAsync();
+    });
+    after(async () => {
+        await blockchainLifecycle.revertAsync();
+    });
+    before(async () => {
+        const accounts = await web3Wrapper.getAvailableAddressesAsync();
+        const makerAddress = accounts[0];
+        contract = await OrderHelperContract.deployFrom0xArtifactAsync(artifacts.OrderHelper, provider, txDefaults);
+
+        const defaultOrderParams = {
+            ...constants.STATIC_ORDER_PARAMS,
+            exchangeAddress: contract.address,
```

Comment:
Tests pass because of this line, but in production this contract's address will be different than the Exchange contract.